documenation.markdown

```diff
@@ -1,24 +1,22 @@
 ---
-layout: default
-title: documentation
+layout: home
+title: Documentation
+permalink: /documentation/
 ---
-
 # Table of Contents
 {:.no_toc}
 
 * replace-me
 {:toc}
 
----
-
 ## Documentation And Information Links
 
 ### The Cylc User Guide
 
 Online copies of the documentation are available here:
 
-* [Multi-Page version](doc/built-sphinx/index.html) (with word search box)
-* [Single-Page version](doc/built-sphinx-single/index.html) - (search with browser Ctrl-F)
+* [Multi-Page version](/../doc/built-sphinx/index.html) (with word search box)
```

Comment:
```suggestion
* [Multi-Page version]({{ 'doc/built-sphinx/index.html' | relative_url}}) (with word search box)
```
This also works in the markdown pages (tested). (And was broken in the original form).